scene/main/node.cpp

```diff
@@ -1316,12 +1316,14 @@ Node *Node::get_node_or_null(const NodePath &p_path) const {
 Node *Node::get_node(const NodePath &p_path) const {
 	Node *node = get_node_or_null(p_path);
 
-	if (p_path.is_absolute()) {
-		ERR_FAIL_COND_V_MSG(!node, nullptr,
-				vformat(R"(Node not found: "%s" (absolute path attempted from "%s").)", p_path, get_path()));
-	} else {
-		ERR_FAIL_COND_V_MSG(!node, nullptr,
-				vformat(R"(Node not found: "%s" (relative to "%s").)", p_path, get_path()));
+	if (!node) {
+		if (p_path.is_absolute()) {
+			ERR_FAIL_V_MSG(nullptr,
+					vformat(R"(Node not found: "%s" (absolute path attempted from "%s").)", p_path, get_path()));
+		} else {
+			ERR_FAIL_V_MSG(nullptr,
+					vformat(R"(Node not found: "%s" (relative to "%s").)", p_path, get_path()));
+		}
 	}
```

Comment:
Alternatively:
```suggestion
	ERR_FAIL_COND_V_MSG(!node, nullptr,
					vformat(p_path.is_absolute()
									? R"(Node not found: "%s" (absolute path attempted from "%s").)"
									: R"(Node not found: "%s" (relative to "%s").)",
							p_path, get_path()));
```
Not the most readable but it's more efficient, as `!node` is called with `unlikely()` in `ERR_FAIL_COND*` which should better optimize the most common path where it's not failing.

Or simpler and more readable:
```suggestion
	if (unlikely(!node)) {
		if (p_path.is_absolute()) {
			ERR_FAIL_V_MSG(nullptr,
					vformat(R"(Node not found: "%s" (absolute path attempted from "%s").)", p_path, get_path()));
		} else {
			ERR_FAIL_V_MSG(nullptr,
					vformat(R"(Node not found: "%s" (relative to "%s").)", p_path, get_path()));
		}
	}
```

Reply:
Thanks! I prefer the simpler/more readable version as both should compile to roughly the same assembly.